app/src/app/components/contents/common/Unauthorized.tsx

```diff
@@ -0,0 +1,10 @@
+export const Unauthorized = () => {
+  return (
+    <div className="container h-[800px] flex items-center justify-center m-auto">
+      <div className="flex flex-col space-y-2 text-center">
+        <h1 className="text-2xl font-semibold tracking-tight text-red-500">401 Unauthorized</h1>
```

Comment:
Putting the 401 here might confuse us, since you haven't necessarily had a 401 back from the API to make this show! And probably wouldn't make much sense to non-tech users. :laughing:  But don't mind!